ios/PurchasesHybridCommon/PurchasesHybridCommonTests/Mocks/MockPurchases.swift

```diff
@@ -558,9 +462,135 @@ class MockPurchases: Purchases {
     var invokedCollectDeviceIdentifiers = false
     var invokedCollectDeviceIdentifiersCount = 0
 
-    override func collectDeviceIdentifiers() {
+    func collectDeviceIdentifiers() {
         invokedCollectDeviceIdentifiers = true
         invokedCollectDeviceIdentifiersCount += 1
     }
     
 }
+
+extension MockPurchases {
```

Comment:
Now it becomes explicit that we're not overriding a bunch of methods, but this ensures that we don't use "the real" implementation when we don't want to.